lib/host/home-bar.js

```diff
@@ -1,49 +0,0 @@
-/* globals scheduler */
-
-window.addEventListener('DOMContentLoaded', function() {
-  var tabs = document.getElementById('tabs');
-  var add = document.getElementById('add');
-
-  window.addEventListener('entering-tabs-view', function() {
-    scheduler.feedback(function() {
-      add.classList.add('show');
-    }, add, 'animationend');
-  });
-
-  window.addEventListener('leaving-tabs-view', function() {
-    scheduler.feedback(function() {
-      add.classList.remove('show');
-    }, add, 'animationend');
-  });
-
-  window.addEventListener('home-button-press', function() {
```

Comment:
we still need something that scrolls back ("closes" the tabs view) when we get a `home-button-press` or a `keyup`.
we can maybe expose a method for this on the `hope-tab-list` and keep a `host.js` file to host the event glue.